src/schema/alerts.ts

```diff
@@ -110,6 +111,10 @@ export const updateAlerts = async (
   const repo = con.getRepository(Alerts);
   const alerts = await repo.findOneBy({ userId });
 
+  if (data.filter === false) {
```

Comment:
@rebelchris though we agreed on using `my_feed` but I realized `alert.filter` is more accurate and much easier to use as it is a boolean than a string. Do you think we can move forward with this?

Reply:
What's the difference between the two?
I meant the one we set once someone created my feed (thought it was that one, but open to the other if it's easier and the same)

Reply:
Perfect then, this was probably the one I meant initially :)